modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbar.ts

```diff
@@ -74,7 +73,7 @@ const register = (editor: Editor, registryContextToolbars: Record<string, Contex
 
   const canLaunchToolbar = () => {
     // If a mobile context menu is open, don't launch else they'll probably overlap. For android, specifically.
-    return !(isTouch() && backstage.isContextMenuOpen());
+    return !editor.removed && !(isTouch() && backstage.isContextMenuOpen());
```

Comment:
This need a changelog entry?

Reply:
Oops, this wasn't supposed to be here yet. That said it won't hurt to leave it here and won't make any functional difference.